ui/admin/app/components/breadcrumbs/container/index.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class BreadcrumbsContainerComponent extends Component {
+  @service('breadcrumbs') breadcrumbsService;
+
+  container = null;
+
+  @action
+  registerContainer(element) {
+    let { itemClass, linkClass } = this.args;
+
+    // A child `ol` is rendered in the Hds::Breadcrumb container and this is
+    // the element that the in-element helper should drop breadcrumbs into.
+    this.container = {
+      element: element.querySelector('ol'),
+      itemClass,
+      linkClass,
```

Comment:
What would we use these classes for? It seems like this is for the package's breadcrumb implementation but we use our own HDS breadcrumbs?

Reply:
good catch, ill remove this.